build-aux/com.obsproject.Studio.json

```diff
@@ -75,15 +75,15 @@
             "buildsystem": "cmake-ninja",
             "builddir": true,
             "config-opts": [
-                "-DCMAKE_BUILD_TYPE=Release",
+                "-DCMAKE_BUILD_TYPE=RelWithDebInfo",
```

Comment:
What's the reason to use a less-optimised build config for Flatpak with this change?